roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -32,5 +34,6 @@
   service:
     name: ceph-osd@{{ item }}
     state: started
+    enabled: true
```

Comment:
One more thing, is it only true for osds? Don't we need the same for other daemons?
Thanks!